README.md

```diff
@@ -1,40 +1,44 @@
-# Sandcastle
+<h1 align="center"> Sandcastle </h1>
+<h2 align="center"> A friendly framework for creating multi-user WebXR apps & multiapps </h2>
 
-![Sandcastle Logo](./sandcastlexr.png)
-
-## A friendly WebXR prototyping engine
+![Sandcastle Samples](./sandcastlexr.png)
 
 ### Features
 
 - Built on vanilla [threeJS](http://threejs.org/)
 
-- Made for WebXR ("out-of-the-box" XR [session&state management](https://github.com/plutovr/sandcastle/wiki#webxr-general) & easy [input event handling](https://github.com/plutovr/sandcastle/wiki#webxr-input))
+- Made for WebXR (ergonomic XR [session & state management](https://github.com/plutovr/sandcastle/wiki#webxr-general) & easy [XR input event handling](https://github.com/plutovr/sandcastle/wiki#webxr-input))
```

Comment:
These links don't work like you might expect them to. I expected them to take me to the section, instead, they take you to the table of contents, and when you click on the "WebXR General" link in the table of context, for example, nothing happens. I noticed a `-1` is appended to the heading that's not in the table of contents.

Reply:
thanks for the heads up! wiki links are _weird_ and constantly change - fixed.